fleet/lib/kube-prometheus-stack/fleet.yaml

```diff
@@ -88,6 +87,13 @@ targetCustomizations:
               - url: https://mimir.ruka.dev.lsst.org/api/v1/push
               - url: https://mimir.antu.ls.lsst.org/api/v1/push
               - url: https://mimir.kueyen.dev.lsst.org/api/v1/push
+            resources:
```

Comment:
Maybe this would be better in a `valuesFile` as I don't think there is a reason to have tu & cl have different values?  That's less of an opportunity for them to drift out of sync in the future.

Reply:
changed! great idea